driver/src/main/scala/com/stratio/sparkta/driver/SparktaClusterJob.scala

```diff
@@ -44,30 +47,35 @@ object SparktaClusterJob extends SparktaSerializer {
     if (checkArgs(args)) {
       Try {
         initSparktaConfig(args(4), args(3))
```

Comment:
Could you change this:
initSparktaConfig(args(4), args(3))

for this:
val ZookeperConfigurationIndex = 4

initiSparktaConfig(args(ZookeperConfigurationIndex), ... )